tests/test_references.py

```diff
@@ -15,7 +15,9 @@
 
 @pytest.mark.parametrize("with_intake", [True, False])
 def test_single(netcdf_local_paths, tmpdir, with_intake):
-    full_paths, items_per_file = netcdf_local_paths
+    full_paths, items_per_file, fnames_by_variable = netcdf_local_paths[:3]
+    if fnames_by_variable:
+        pytest.skip("This does not test merging operations.")
```

Comment:
Now that we have #174 merged _and_ we have FilePatterns as fixtures, we should refactor these test to use the FilePattern fixtures (rather than `netcdf_local_paths`).

Reply:
We might have to define a new FilePattern fixture that contains just a single file for this test. That should be done in `conftest.py`.